src/singletons/Settings.hpp

```diff
@@ -364,6 +366,7 @@ class Settings : public ABSettings, public ConcurrentSettings
 
     /// Logging
     BoolSetting enableLogging = {"/logging/enabled", false};
+    BoolSetting enableCustomLogging = {"/logging/customLogging", false};
```

Comment:
Setting name & path should be adapted to something more along the lines of "onlyLogListedChannels" or w/e phrasing makes most sense